aexpect/client.py

```diff
@@ -166,7 +167,12 @@ def __init__(self, command=None, a_id=None, auto_close=False, echo=False,
 
         # Start the server (which runs the command)
         if command:
-            helper_cmd = utils_path.find_command('aexpect-helper')
+            # try to find python specific version of aexpect-helper first, then try unversioned
+            helper_noversion = 'aexpect-helper'
+            helper_versioned = helper_noversion + '-{0.major}.{0.minor}'.format(sys.version_info)
```

Comment:
A style suggestion (shorter lines, although still larger than 79 chars):

```
            helper_versioned = '{0}-{1.major}.{1.minor}'.format(helper_noversion,
                                                                sys.version_info)
```

Reply:
Updated as suggested, then I let `autopep8 ` decide on the line wrapping...